packages/experiments/src/components/Stack/StackItem/StackItem.tsx

```diff
@@ -5,17 +5,16 @@ import { IStackItemComponent, IStackItemProps, IStackItemSlots } from './StackIt
 import { styles } from './StackItem.styles';
 
 const view: IStackItemComponent['view'] = props => {
-  const childNodes: React.ReactElement<{}>[] = React.Children.toArray(props.children) as React.ReactElement<{}>[];
-  const first = childNodes[0];
-  if (!first) {
+  const { children } = props;
+  if (React.Children.count(children) < 1) {
```

Comment:
Nice change! Can `props.children` ever be negative? If not, perhaps this conditional could be:

```suggestion
  if (React.Children.count(children) === 0) {
```

Reply:
Since the only valid values we want to allow are `> 1` I tend to prefer being defensive and excluding all unnecessary scenarios, even if `React.Children.count` *currently* returns 0. Probably a pretty minor concern in this specific example, but assuming there isn't a perf difference, I like being more defensive and it's why I suggested `< 1`.